projects/admin/src/app/record/detail-view/holding-detail-view/serial-holding-detail-view/serial-holding-detail-view.component.html

```diff
@@ -0,0 +1,144 @@
+<!--
+  RERO ILS UI
+   Copyright (C) 2019 RERO
+  
+   This program is free software: you can redistribute it and/or modify
+   it under the terms of the GNU Affero General Public License as published by
+   the Free Software Foundation, version 3 of the License.
+  
+   This program is distributed in the hope that it will be useful,
+   but WITHOUT ANY WARRANTY; without even the implied warranty of
+   MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+   GNU Affero General Public License for more details.
+  
+   You should have received a copy of the GNU Affero General Public License
+   along with this program.  If not, see <http://www.gnu.org/licenses/>.
+-->
+<h1 class="mb-5">
+  {{ 'Holding' | translate }} : {{ 'Receive an issue' | translate }}
+</h1>
+
+<h3>
+  {{ 'Issues' | translate }}
+  <button class="btn btn-sm btn-outline-primary float-right"
+          [routerLink]="['/records', 'items', 'new']"
+          [queryParams]="{
+              'holding': holding.id,
+              'irregular': true,
+              'redirectTo': 'records/holdings/detail/' + holding.id
+          }"
+  >
+    <i class="fa fa-plus-square-o"></i>
+    {{ 'Add irregular issue' | translate }} ...
+  </button>
+</h3>
+<div class="card mt-2">
+  <div class="card-header">
+    <div class="row">
+      <div class="col-sm-4" translate>Numbering</div>
+      <div class="col-sm-1 text-center" translate>Status</div>
+      <div class="col-sm-2" translate>Call number</div>
+      <div class="col-sm-3" translate>Reception date</div>
+      <div class="col-sm-2">&nbsp;</div>
+    </div>
+  </div>
+  <div class="card-body">
+
+    <!-- SHOW MORE PREDICTIONS ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~ -->
+    <div class="row mt-1 mb-2">
+      <a href="javascript:void(0)" (click)="showMore('prediction', 3)" class="col-sm-12">
```

Comment:
I think it should be a better approach than `javascript:void(0)`

Reply:
I change to use `[routerLink]=""` ; it seems a better Angular approach